data/xml/2019.rocling.xml

```diff
@@ -0,0 +1,362 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<collection id="2019.rocling">
+  <volume id="1" ingest-date="2021-06-16">
+    <meta>
+      <booktitle>Proceedings of the 31st Conference on Computational Linguistics and Speech Processing (ROCLING 2019)</booktitle>
+      <editor><first>Chen-Yu</first><last>Chiag</last></editor>
+      <editor><first>Min-Yuh</first><last>Day</last></editor>
+      <editor><first>Jen-Tzung</first><last>Chien</last></editor>
+      <publisher>The Association for Computational Linguistics and Chinese Language Processing (ACLCLP)</publisher>
+      <address>New Taipei City, Taiwan</address>
+      <month>October</month>
+      <year>2019</year>
+      <url hash="7cb428a0">2019.rocling-1</url>
+    </meta>
+    <frontmatter>
+      <url hash="f50c4de0">2019.rocling-1.0</url>
+    </frontmatter>
+    <paper id="1">
+      <title>基於特徵粒度之訓練策略於中文口語問答系統之應用(A Feature-granularity Training Strategy for <fixed-case>C</fixed-case>hinese Spoken Question Answering) [In <fixed-case>C</fixed-case>hinese]</title>
```

Comment:
I suggest that we remove the `[In Chinese]` from the title, and just add a `<language>zhe</language>` tag instead.

Reply:
(Do this for all papers)

Reply:
okay got it